src/pages/openPRs.tsx

```diff
@@ -0,0 +1,41 @@
+import { FunctionComponent, useState, useEffect } from 'react';
+import fetch from '../helperFunctions/fetch';
+import Layout from '../components/Layout';
+import Navbar from '../components/navbar';
+import PullRequest from '../components/pullRequests';
+
+const openPRs: FunctionComponent = () => {
+    const url = 'https://staging-api.realdevsquad.com/pullrequests/open';
+    const [state, setState] = useState(null);
+
+    useEffect(() => {
+        (async () => {
+            const response = await fetch(url);
+            setState(response.data.pullRequests);
+        })();
+    }, []);
+
+    const getPRs = () => {
+        return state.map((pullRequest) => {
+            return (
+                <PullRequest key={pullRequest.title} 
+                             title={pullRequest.title} 
+                             username={pullRequest.username}
+                             createdAt={pullRequest.createdAt}
+                             updatedAt={pullRequest.updatedAt}
+                             url={pullRequest.url} />
```

Comment:
NIT: Can we fix indentation?

Reply:
![image](https://user-images.githubusercontent.com/11243646/103903961-98d89200-5122-11eb-8963-18c30c68a09c.png)

is this fine?